src-cpp/rdkafkacpp.h

```diff
@@ -81,13 +81,20 @@ typedef SSIZE_T ssize_t;
 
 /**@endcond*/
 
+#ifdef _MSC_VER    /* Visual Studio */
+#  pragma warning(disable : 4273)
+#endif
+
 extern "C" {
         /* Forward declarations */
         struct rd_kafka_s;
         struct rd_kafka_topic_s;
         struct rd_kafka_message_s;
         struct rd_kafka_conf_s;
         struct rd_kafka_topic_conf_s;
+
+        RD_EXPORT void *rd_kafka_mem_malloc(struct rd_kafka_s *rk, size_t size);
```

Comment:
I'd rather see a proper C++ wrapper API, we'll want to avoid conflating the two APIs unless necessary.